interview-questions.md

```diff
@@ -8,33 +8,33 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX connects JS to HTML
```

Comment:
One liner answers will not impress an interviewer.  Try expanding your answers by providing an example of a time you have used JSX.